x-pack/plugin/runtime-fields/qa/core-with-search/build.gradle

```diff
@@ -0,0 +1,55 @@
+apply plugin: 'elasticsearch.yaml-rest-test'
+
+dependencies {
+  yamlRestTestImplementation xpackProject("plugin:runtime-fields:qa")
+}
+
+restResources {
+  restApi {
+    includeXpack 'async_search', 'graph', '*_point_in_time'
+  }
+  restTests {
+    includeCore '*'
+    includeXpack 'async_search', 'graph'
+  }
+}
+
+testClusters.yamlRestTest {
+  testDistribution = 'DEFAULT'
+  setting 'xpack.license.self_generated.type', 'trial'
+}
+
+yamlRestTest {
+  systemProperty 'tests.rest.suite',
+    [
+      'async_search',
+//      'field_caps',    These two don't support runtime fields on the request. Should they?
+//      'graph',
+//      'msearch',       This one just doesn't have infrastructure to hack the runtime fields into place
+      'search',
+      'search.aggregation',
+      'search.highlight',
+      'search.inner_hits',
+      'search_shards',
+      'suggest',
+    ].join(',')
+  systemProperty 'tests.rest.blacklist',
+    [
+      /////// TO FIX ///////
+      'search.highlight/40_keyword_ignore/Plain Highligher should skip highlighting ignored keyword values', // The plain highlighter is incompatible with runtime fields. Worth fixing?
```

Comment:
We may find that #63572 helps here

Reply:
Yeah. I've not been pushing on these particularly hard. I probably should be.